README.md

```diff
@@ -92,5 +92,5 @@ However, such problems are best solved with social norms, not with software
 licenses. Any attempt to complicate the software license of Zig would risk
 compromising the value Zig provides to users.
 
-Therefore, Zig is available under the MIT (Expat) License, and comes with a
+Therefore, Zig is available under the MIT (Expat) License (except where otherwise noted), and comes with a
```

Comment:
Maybe "except where otherwise noted __to be even less restrictive__"? Just to make sure it doesn't sound unintentionally scarier than before.

Reply:
Do you mean less permissive? We distribute gnu libc (which is LGPL) and images that are CC-BY-SA and we do not have a right to re-license these works under MIT.

Reply:
Oh, I was just looking at the CC licenses in this PR, which I thought were more permissive than MIT (which they are not - MIT allows relicensing from what I can see? - CC-SA requires providing the same license); I wasn't even thinking about glibc and dependencies.
Right, in that case ignore my comment.